app/activeplan/homepage/get.controller.js

```diff
@@ -1,31 +1,45 @@
 const { logger } = require('../../../common/logging/logger')
-const { getSentencePlan } = require('../../../common/data/sentencePlanningApi')
+const { getSentencePlan, getSentencePlanMeetings } = require('../../../common/data/sentencePlanningApi')
 const { getCommentText } = require('../../../common/utils/getCommentText')
 
 const getHomepage = async (req, res) => {
   const {
     session: { planStarted = false, 'x-auth-token': token },
     params: { planId, id },
   } = req
-  try {
-    delete req.session.planStarted
 
-    const { comments } = await getSentencePlan(planId, token)
+  let sentencePlan
+  let meetings
+
+  delete req.session.planStarted
 
-    return res.render(`${__dirname}/index`, {
-      planId,
-      id,
-      planStarted,
-      contactArrangements: getCommentText(comments, 'LIAISON_ARRANGEMENTS'),
-      diversity: getCommentText(comments, 'YOUR_RESPONSIVITY'),
-      decisions: getCommentText(comments, 'YOUR_SUMMARY'),
-      needToKnow: getCommentText(comments, 'THEIR_RESPONSIVITY'),
-      comments: getCommentText(comments, 'THEIR_SUMMARY'),
-    })
+  // get the sentence plan
+  try {
+    sentencePlan = await getSentencePlan(planId, token)
   } catch (error) {
-    logger.error(`Could not display active sentence plan ${planId} for offender ${id}, error: ${error}`)
+    logger.error(`Could not retrieve active sentence plan ${planId} for offender ${id}, error: ${error}`)
     return res.render('app/error', { error })
   }
+
+  // get review meetings
+  try {
+    meetings = await getSentencePlanMeetings(id, token)
+  } catch (error) {
+    logger.error(`Could not retrieve meetings for offender ${id}, error: ${error}`)
+    return res.render('app/error', { error })
+  }
```

Comment:
Because the error is purely attached to the async, I'd argue we can just `.catch` them?
```
const sentencePlan = await getSentencePlan(planId, token).catch(error => {
  logger.error(`Could not retrieve active sentence plan ${planId} for offender ${id}, error: ${error}`
  return res.render('app/error', { error })
})
const meetings = await getSentencePlanMeetings(id, token).catch(error => {
  logger.error(`Could not retrieve meetings for offender ${id}, error: ${error}`)
  return res.render('app/error', { error })
})
```